sample-apps/donet-sample-app/donet-sample-app/Controllers/AppController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Amazon.S3;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Net.Http;
+
+namespace donet_sample_app.Controllers
+{
+
+    var meter = new Meter("MyApplication");
+
+    var counter = meter.CreateCounter<int>("Requests");
+    var histogram = meter.CreateHistogram<float>("RequestDuration", unit: "ms");
```

Comment:
Is this being used at all?

Reply:
Fixed the issue as I was experimenting with `histogram.observe`